neonvm/runner/main.go

```diff
@@ -1217,6 +1229,16 @@ func defaultNetwork(logger *zap.Logger, cidr string, ports []vmv1.Port) (mac.MAC
 		return nil, err
 	}
 
+	f, err := os.OpenFile("/etc/hosts", os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
+	if err != nil {
+		return nil, err
+	}
+	defer f.Close()
+	record := fmt.Sprintf("%v neonvm-br\n", ipVm)
```

Comment:
why `-br`?

Reply:
To show it's on the bridge network between the pod and the VM (it's named `br-def`). Better names are welcome.

Reply:
Maybe `neonvm-local`, dunno

Reply:
IMO: `guest` or `neonvm-guest` or something. That way, the ssh looks like
```sh
kubectl exec -it <pod name> -- ssh guest
# ... or ...
kubectl exec -it <pod name> -- ssh neonvm-guest
```

basically: the thing we're connecting to isn't the bridge network itself, it's the VM.

maybe even just `ssh vm` ?

Reply:
I'd love to go to with something concise (like `vm`). The caveat is that we cover this DNS name. For example, in the future we can't have a k8s service named `vm` and access that through DNS.

Reply:
That's a good point. Maybe let's ask in #dev for feedback? this will impact anyone who connects into a VM, so probably worthwhile. Can just be a simple poll

Reply:
I think it's overkill to ask in #dev. Let's go with something safe like `guest-vm`.

Reply:
Sure, `guest-vm` works.
Part of the reason I suggest #dev is that it also adds visibility onto upcoming improvements. Many people care about this, would have useful feedback, and would be excited to hear about the work you're doing.
But ultimately I'll leave it up to you.

Reply:
I'll announce once we deployed on staging.

Reply:
renamed in dbd1f507ad4f095903c1fa5fa9cffd719232649b